src/lib/helpers/isRouteInPrerenderManifest.js

```diff
@@ -1,20 +1,21 @@
 const getPrerenderManifest = require('./getPrerenderManifest')
-const i18n = require('./getI18n')()
+const getI18n = require('./getI18n')
 
-const { routes, dynamicRoutes } = getPrerenderManifest()
-const { defaultLocale, locales } = i18n
+// Return true if the route is defined in the prerender manifest
+const isRouteInPrerenderManifest = async (route) => {
+  const i18n = await getI18n()
+  const { defaultLocale, locales } = i18n
+  const { routes, dynamicRoutes } = await getPrerenderManifest()
 
-const isRouteInManifestWithI18n = (route) => {
-  let isStaticRoute = false
-  Object.entries(routes).forEach(([staticRoute, { srcRoute }]) => {
-    // This is because in i18n we set the nakedRoute to be the srcRoute in the manifest
-    if (route === srcRoute) isStaticRoute = true
-  })
-  return isStaticRoute || route in dynamicRoutes
-}
+  const isRouteInManifestWithI18n = () => {
+    let isStaticRoute = false
+    Object.entries(routes).forEach(([staticRoute, { srcRoute }]) => {
+      // This is because in i18n we set the nakedRoute to be the srcRoute in the manifest
+      if (route === srcRoute) isStaticRoute = true
+    })
+    return isStaticRoute || route in dynamicRoutes
+  }
 
-// Return true if the route is defined in the prerender manifest
-const isRouteInPrerenderManifest = (route) => {
   if (i18n.defaultLocale) return isRouteInManifestWithI18n(route)
```

Comment:
```suggestion
  if (i18n.defaultLocale) return isRouteInManifestWithI18n()
```

doesnt rly matter tho